.github/actions/build-and-deploy/action.yml

```diff
@@ -0,0 +1,40 @@
+name: "Build and deploy neon-is-group-loader module"
```

Comment:
remove "deploy" (this action doesn't deploy)

Reply:
Changed to "Build and push latest to nonprod"